cogs/general.py

```diff
@@ -21,6 +21,11 @@ async def ping(self, inter):
         )
         await inter.response.send_message(embed=embed)
 
+    @slash_command(name="invite", description="invite bot on new serv")
+    async def invite(self, inter):
+        invite = "https://discord.com/api/oauth2/authorize?client_id=1050387869996425276&permissions=8&scope=bot%20applications.commands"
```

Comment:
Add this to .env file to prevent further developments needs to edit source code in order to edit this setting ;)